hwdb.d/60-sensor.hwdb

```diff
@@ -693,7 +693,7 @@ sensor:modalias:acpi:SMO8500*:dmi:bvnAmericanMegatrendsInc.:bvr5.6.5:bd07/25/201
 
 # One-netbook OneMix 2s
 # OneMix 2s has no product name filled, matching entire dmi-alias
-sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvr5.12:bd10/26/2018:br5.12:svnDefaultstring:pnDefaultstring:pvrDefaultstring:rvnDefaultstring:rnDefaultstring:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:*
+sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvr5.12:bd10/26/2018:br5.12:svnDefaultstring:pnDefaultstring:pvrDefaultstring:skuDefaultstring:rvnDefaultstring:rnDefaultstring:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
```

Comment:
By removing the old /sys/class/dmi/id/modalias value you will break this for older kernels, instead of outright replacing it, please add a '*' at the place where newer kernels insert the skuDefaultstring: into the DMI modalias so that it will work with both old and new kernels.

Reply:
So like this?

`sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvr5.12:bd10/26/2018:br5.12:svnDefaultstring:pnDefaultstring:pvrDefaultstring:*:rvnDefaultstring:rnDefaultstring:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:`

Reply:
Almost, you are adding not only a `*` but also a `:` extra, so the old string will still not match.

Also I just noticed that you are also dropping the `*` at the end, that is there deliberately for if the udev rules start adding extra info at the end for narrower matches. So we must keep the `*` at the end.

So I believe what you want is this: `sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvr5.12:bd10/26/2018:br5.12:svnDefaultstring:pnDefaultstring:pvrDefaultstring:*rvnDefaultstring:rnDefaultstring:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:*`

Reply:
Thanks for helping! I created new commit for this.